Example/Podfile

```diff
@@ -1,8 +1,9 @@
-source 'https://cdn.cocoapods.org'
 source 'https://github.com/bitmovin/cocoapod-specs.git'
+source 'https://github.com/CocoaPods/Specs.git'
+
 def shared_pods
   pod 'BitmovinAdobeAnalytics', path: '../'
-  pod 'BitmovinPlayer', '2.57.0'
+  pod 'BitmovinPlayer', '3.6.0'
```

Comment:
Should we use `~> 3.6` here?

Reply:
This Pod file is for Example application and I have added the version which was tested. Also changing to ~> 3.6 will not help with versions >= 3.7.0

Reply:
> This Pod file is for Example application and I have added the version which was tested.

Fair enough

> Also changing to ~> 3.6 will not help with versions >= 3.7.0

https://guides.cocoapods.org/using/the-podfile.html says:

> '~> 0.1.2' Version 0.1.2 and the versions up to 0.2, not including 0.2 and higher
> '~> 0.1' Version 0.1 and the versions up to 1.0, not including 1.0 and higher
> '~> 0' Version 0 and the versions up to 1.0, not including 1.0 and higher

So to my understanding, `~> 3.6` would mean `>= 3.6.0 && <4.0.0`

Reply:
You are right. We added ~>3.0 to the Podspec so that application can use any 3.x.x version.